### DISCUSSION TITLE
"loop" not included with install

### DISCUSSION THREAD

**Discussion by TheColetrain** [Ride-The-Lightning/RTL#1376] [Q&A]
Hello.  I am a Noob, to be clear. 

I recently installed Umbrel, and using Umbrel, I installed RTL.  
my RTL does not have "loop"  
currently under "services" only "Boltz" exists.  "Loop"  is not there.   (Attached shows both, but I only have one) 

![loop](https://github.com/Ride-The-Lightning/RTL/assets/8950046/461257c9-151a-4e38-a1e2-6c46881c37c4)

My biggest problem is that I am just getting started and all guides and youtubes are on "loop" and there are few guides on "boltz"  

Can I either have a good guide on Bolz, or a good guide on how to install loop?

Thank you

**Comment by saubyk** [comment]
Hi @TheColetrain Loop service has been removed from the Umbrel instance of RTL. So you'll not be able to access it from RTL on Umbrel

If you're interested in using the Loop service, you can install the Lightning Terminal on Umbrel and use it from there

**Reply by TheColetrain** [reply]
Step one. THANK YOU for clearing that up!

Two: I did notice the loop on Terminal. Do you know of a guide or video for that. (Have not started my googling on that, but in case you knew off hand.)

**Reply by saubyk** [reply]
in case you still need it: https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/loop
